pexpect/pxssh.py

```diff
@@ -294,8 +294,15 @@ def login (self, server, username, password='', terminal_type='ansi',
         session to do so. Setting this option to `False` and not having an active session
         will trigger an error.
         
-        Set ``ssh_key`` to `True` to force passing the current SSH authentication socket to the
+        Setting ``ssh_key`` to a file path to an SSH private key will use that SSH key
```

Comment:
I'd like the docstring to be consistent about using either "Set ... to ..." or "Setting ... will ...". I don't have a strong preference which (though it's easier to change the bits you've just added), but it should use the same one throughout.

Reply:
Sure thing.